dev/unix/volta-install.sh

```diff
@@ -133,11 +134,10 @@ parse_os_info() {
       if [ "$exit_code" != 0 ]; then
         return "$exit_code"
       fi
-
-      echo "linux-openssl-$parsed_version"
+      echo "linux-openssl-$parsed_version-$arch_str"
```

Comment:
For backwards compatibility, can we handle the `$arch_str` in the same way as the mac case? Where `x86_64` is unadorned, while the `arm64` one gets an extra suffix? That would make sure that the install script still works for installing older versions (where the existing files don't have the `-x86_64` suffix). While I don't think it's super common, we do technically support a user passing `--version 1.0.4` to the script to get an older versions of Volta.